### ISSUE TITLE
IRuby integration is broken

### ISSUE THREAD

**Issue by mrkn** [mrkn/matplotlib.rb#18]
Using with matplotlib 3.6.0, the following error occurs in `Matplotlib::IRuby.activate`.

```
PyCall::PyError: <class 'AttributeError'>: module 'matplotlib_rb.backend_inline' has no attribute 'FigureCanvas'
  File "/opt/conda/lib/python3.10/site-packages/matplotlib/pyplot.py", line 266, in switch_backend
    canvas_class = backend_mod.FigureCanvas

/home/jovyan/.local/share/gem/ruby/3.1.0/gems/matplotlib-1.2.0/lib/matplotlib/iruby.rb:201:in `switch_backend'
/home/jovyan/.local/share/gem/ruby/3.1.0/gems/matplotlib-1.2.0/lib/matplotlib/iruby.rb:201:in `activate_matplotlib'
/home/jovyan/.local/share/gem/ruby/3.1.0/gems/matplotlib-1.2.0/lib/matplotlib/iruby.rb:152:in `enable_matplotlib'
/home/jovyan/.local/share/gem/ruby/3.1.0/gems/matplotlib-1.2.0/lib/matplotlib/iruby.rb:111:in `activate'
(irb):1:in `<top (required)>'
```

**Comment by kojix2** [comment]
Here is a minimal reproducible example:

```ruby
require "iruby"
require "matplotlib/iruby"

# `activate_matplotlib` is a private method
Matplotlib::IRuby.send("activate_matplotlib", "module://matplotlib_rb.backend_inline")
```

In my environment, replacing FigureCanvasAgg with FigureCanvas worked.

https://github.com/mrkn/matplotlib.rb/blob/3fd731b2c5cd4897f3f5e21105bad36627f8fa88/lib/matplotlib/python/matplotlib_rb/backend_inline.py#L1

**Comment by mrkn** [comment]
@kojix2 Thank you for the information.

**Comment by mrkn** [comment]
I confirmed that https://github.com/matplotlib/matplotlib/pull/23090 is the reason.  It changed the `pyplot.switch_backend` behavior to refer to `FigureCanvas`.

**Comment by mrkn** [comment]
Fixed by 1b7328ef578da953e4c54e64d4c465d09cb5d138

**Comment by kozo2** [comment]
Thanks for https://github.com/mrkn/matplotlib.rb/commit/1b7328ef578da953e4c54e64d4c465d09cb5d138 .
Now I have succeeded in embedding the matplotlib.rb image in the ipynb output cell.
![image](https://user-images.githubusercontent.com/12192/226088027-2b714a3c-e0c8-4152-827d-0f11c464a37c.png)

That is what I needed to do! Thanks!
